### PR TITLE
Ami readmi

### PR DESCRIPTION
**Pull request by ggxed**

### CODE REVIEW

File: ami/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Team Edition Helm chart for AWS AMI
+
+#### Minimum requirements:
+
+* t2.large instance
+* 2 CPUs
+* 8Gb RAM
```
Comment (serge-rider) [inline ami/README.md:7]: 16gb

File: ami/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
```
Comment (serge-rider) [inline ami/README.md:9]: Please add information how to deploy AMI

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: ami/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+## Team Edition Helm chart for AWS AMI
+
+#### Minimum requirements:
+
+* 4 CPUs
+* 16Gb RAM
```
Comment (arhayka) [inline ami/README.md:6]: * 16GB RAM

File: ami/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* 100G Storage (SSD recommended)
```
Comment (arhayka) [inline ami/README.md:7]: * 100GB Storage (SSD recommended)

File: ami/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### How to deploy AMI in AWS
+
+- Go to AWS Marketplace
+- Choose [DBeaver Team Edition](https://aws.amazon.com/marketplace/pp/prodview-kijugxnqada5i?sr=0-2&ref_=beagle&applicationId=AWSMPContessa)
+- Use recommended instance resources for the best experience with this product
+- For your secure not recommended make the service public
```
Comment (arhayka) [inline ami/README.md:14]: - For security reasons, it is not recommended to make the service public

File: ami/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+   - Certificate: `/fullchain.pem`  
+   - Private Key: `/privkey.pem`
+- Change `CLOUDBEAVER_DOMAIN=localhost` to your domain in .env file
+- Enter `dbeaver-te stop` and `dbeaver-te start` to accept new config
+
+
+### Version update procedure.
+
+- Enter `dbeaver-te update list`
+- Choose the version which you want to update
```
Comment (arhayka) [inline ami/README.md:35]: - Choose the version you want to update

File: README.md [hunk 7]
```diff
@@ -10,6 +10,7 @@ or in a cloud.
 
 - [Docker compose](compose) - the simplest way to deploy and run the server on the local machine
 - [Kubernetes](k8s) - if you prefer to run everything with docker orchestration 
+- [AWS AMI](ami) - if you want use AWS for deploy
```
Comment (arhayka) [inline README.md:13]: - [AWS AMI](ami) - if you want to use AWS for deployment

**Review by arhayka — COMMENTED** [review]
Some micro changes :)

**Pull request closed without merging**